src/lwc/utilInput/utilInput.html

```diff
@@ -57,6 +57,22 @@
             </c-ge-form-field-picklist>
         </template>
 
+        <template if:true={isTrueFalsePicklist}>
+            <c-ge-form-field-picklist class-name="slds-m-around_none"
+                                      data-id='inputComponent'
+                                      label={uiLabel}
+                                      required={isRequired}
+                                      field-name={sourceFieldApiName}
+                                      object-name={sourceObjectDescribe.apiName}
+                                      object-describe-info={sourceObjectDescribe}
+                                      onchange={handleChangeCombobox}
+                                      value={fieldValue}
+                                      variant={variant}
+                                      tab-index={tabIndex}
```

Comment:
This tab-index property causes a double tab stop on the picklist and should be removed if this component is kept in the template.